rust/README.md

```diff
@@ -2,8 +2,13 @@
 
 This grammar is based on official language reference.
 
-This grammar only implemented v2018+ stable feature at the moment, not exactly match with official grammar (may differ at some edge case like isolated \r).
-
 Licensed under MIT
 
 Entry rule is `crate`, you need run test via command line at the moment: `grun Rust crate examples/*.rs`.
+
+Last updated for rust v1.49.0
+
+## Known limitation
+- Only v2018+ stable feature is implemented.
```

Comment:
Please always insert a line break between header and paragraph, see [MD022 - Headers should be surrounded by blank lines](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md022---headers-should-be-surrounded-by-blank-lines) rule.

```suggestion
## Known limitation

- Only v2018+ stable feature is implemented.
```